p256/fiat_p256.mli

```diff
@@ -1,3 +1,14 @@
+(** Diffie-Hellman key exchange over P-256 (secp256r1) curve.
+    This implementation uses C code from {{https://github.com/mit-plv/fiat-crypto}
+    Fiat-crypto project}.
```

Comment:
IIRC the syntax for ocamldoc hyperlnks is `{{:https://gtihub.com/...}name}`, i.e. there's a `:` missing